python/hail/dataset.py

```diff
@@ -253,6 +253,27 @@ def aggregate_intervals(self, input, condition, output):
     def annotate_alleles_expr(self, condition, propagate_gq=False):
         """Annotate alleles with expression.
 
+        **Notes**
```

Comment:
@tpoterba says we will be reworking this command soon, so this short description is fine.